src/test/org/elmlang/intellijplugin/features/intention/imports/ElmImportQuickFixTest.java

```diff
@@ -0,0 +1,89 @@
+package org.elmlang.intellijplugin.features.intention.imports;
+
+import com.intellij.testFramework.fixtures.LightPlatformCodeInsightFixtureTestCase;
+
+public class ElmImportQuickFixTest extends LightPlatformCodeInsightFixtureTestCase {
```

Comment:
Hmm... never seen that before. I *do* see the first error (log4j complaining about appenders), but never seen the `NoClassDefFoundError`.

Strangely, I get a *different* error consistently, but the tests still run fine:
```
ERROR: Local history is broken(version:6,current timestamp:Sep 3, 2016 5:28:23 AM,storage timestamp:Aug 28, 2016 5:42:39 AM,vfs timestamp:Aug 28, 2016 5:42:39 AM)
null
java.io.IOException: Recursive records found
	at com.intellij.history.core.ChangeListStorageImpl.doReadPrevSafely(ChangeListStorageImpl.java:280)
	at com.intellij.history.core.ChangeListStorageImpl.findFirstObsoleteBlock(ChangeListStorageImpl.java:271)
	at com.intellij.history.core.ChangeListStorageImpl.purge(ChangeListStorageImpl.java:237)
	at com.intellij.history.core.ChangeList.purgeObsolete(ChangeList.java:175)
	at com.intellij.history.integration.LocalHistoryImpl.disposeComponent(LocalHistoryImpl.java:147)
	at com.intellij.history.integration.LocalHistoryImpl$1.run(LocalHistoryImpl.java:70)
	at com.intellij.openapi.util.ShutDownTracker.run(ShutDownTracker.java:60)
	at java.lang.Thread.run(Thread.java:745)
```

Here's my IntelliJ info:
IntelliJ IDEA 2016.1.3
Build #IC-145.1617, built on June 3, 2016
JRE: 1.8.0_76-release-b198 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o

As for the Community Edition source code, I have `d742435` checked out as HEAD. Maybe I did this wrong? (it's my first time doing any IntelliJ plugin development).